pkg/ruler/base/ruler_test.go

```diff
@@ -1665,7 +1667,7 @@ func TestSendAlerts(t *testing.T) {
 					Annotations:  []labels.Label{{Name: "a2", Value: "v2"}},
 					StartsAt:     time.Unix(2, 0),
 					EndsAt:       time.Unix(3, 0),
-					GeneratorURL: "http://localhost:9090/graph?g0.expr=up&g0.tab=1",
+					GeneratorURL: fmt.Sprintf("http://localhost:9090/explore?left={\"queries\":[%s]}", escapedExpression),
```

Comment:
Tiny nit: it's mildly confusing here to keep using `:9090` since that's the Prometheus port